src/SchemaUrl.php

```diff
@@ -36,28 +41,28 @@ public static function fromSchema($format, SchemaInterface $schema) {
    *
    * @param string $format
    *   The format or type of schema.
-   * @param string $entity_type
+   * @param string $describes
+   *   The format being described.
+   * @param string $entity_type_id
    *   The entity type.
    * @param string $bundle
    *   The entity bundle.
    *
    * @return \Drupal\Core\Url
    *   The schema resource Url object.
    */
-  public static function fromOptions($format, $entity_type, $bundle = NULL) {
-    $route = empty($bundle) ? 'rest.schemata_entity_base.GET.' . $format
-      : 'rest.schemata_entity_bundle.GET.' . $format;
-
-    $parameters = [
-      'entity_type' => $entity_type,
-    ];
-    if (isset($bundle)) {
+  public static function fromOptions($format, $describes, $entity_type_id, $bundle = NULL) {
```

Comment:
Not sure if it's coming from here or something about the route itself, but an unneeded querystring parameter of entity_type is being appended to the URL.